toucan_connectors/wootric/wootric_connector.py

```diff
@@ -28,7 +28,12 @@
 
 def batch_fetch(urls):
     """fetch asyncrhonously `urls` in a single batch"""
-    loop = asyncio.get_event_loop()
+    # loop = asyncio.get_event_loop()
+    try:
+        loop = asyncio.get_event_loop()
+    except RuntimeError:
```

Comment:
I'm not an `asyncio` expert, why would there be a `RuntimeError`? I thought one was created when there was none yet